assets/stylesheets/style.css

```diff
@@ -49,10 +50,12 @@ div.offset-s {
 
 .rush-x {
 	width: 100%;
+	max-width: 700px;
```

Comment:
max-width is actually not allowed in `ebook` formats whcih is why I removed it originally.